Integrations/python/deephaven/learn/gather/__init__.py

```diff
@@ -0,0 +1,123 @@
+# 
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent Pending
+# 
+"""
+Utilities for gathering Deephaven table data into Python objects
+"""
+
+import numpy as np
+import enum
+import jpy
+import wrapt
+
+# None until the first _defineSymbols() call
+_gatherer = None
+
+def _defineSymbols():
+    if not jpy.has_jvm():
+        raise SystemError("No java functionality can be used until the JVM has been initialized through the jpy module")
+
+    global _gatherer
+    global Layout
+
+    if _gatherer is None:
+        _gatherer = jpy.get_type("io.deephaven.integrations.learn.gather.NumPy")
+    
+# A class defining the memory layout of an array
+class MemoryLayout(enum.Enum):
+    """
+    Class with boolean values for memory layout of an array
+
+    :param enum.Enum: Base class for creating enumerated constants
+    """
+    ROW_MAJOR = True
+    COLUMN_MAJOR = False
+    C = True
+    FORTRAN = False
+    def __init__(self, is_row_major):
+        self.is_row_major = is_row_major
+    @classmethod
+    def has_value(cls, value):
```

Comment:
```suggestion
    FORTRAN = False
    
    def __init__(self, is_row_major):
        self.is_row_major = is_row_major
        
    @classmethod
    def has_value(cls, value):
```
Whitespace for readability.

Reply:
Document `has_value`